pkg/cri/server/container_create_linux_test.go

```diff
@@ -888,6 +888,8 @@ func TestUserNamespace(t *testing.T) {
 			err: true,
 		},
 	} {
+		desc := desc
+		test := test
```

Comment:
Hmm, is this enough? Because this var is global, for example: https://github.com/containerd/containerd/pull/7880/files#diff-da0f598f71bf7a8f8b340f4e63b7151bfcf9c1d366aa6fb2ed1f292a84edda5eL822

@fuweid I guess what you suggested is not enough to avoid the t.Parallel() issues (that other tests in this file also have). Should I try to solve those too here?

Reply:
Yes! When t.Run with t.Parallel, different cases might share the same data.
The change looks good to me.